From https://github.com/jgm/pandoc/commit/532bbc875bbcb52ddbaa02600bb856d7e32b795e
From: John MacFarlane <jgm@berkeley.edu>
Date: Mon, 17 Jun 2024 19:57:33 -0700
Subject: [PATCH] Use latest commonmark-extensions.

This fixes `auto_identifiers` and `ascii_auto_identifiers`
extensions so that they substitute aliases for emojis,
as documented.

See #9876.
--- a/pandoc.cabal
+++ b/pandoc.cabal
@@ -482,7 +482,7 @@ library
                  case-insensitive      >= 1.2      && < 1.3,
                  citeproc              >= 0.8.1    && < 0.9,
                  commonmark            >= 0.2.6    && < 0.3,
-                 commonmark-extensions >= 0.2.5.4  && < 0.3,
+                 commonmark-extensions >= 0.2.5.5  && < 0.3,
                  commonmark-pandoc     >= 0.2.2.1  && < 0.3,
                  containers            >= 0.6.0.1  && < 0.8,
                  crypton-connection    >= 0.3.1    && < 0.5,
--- a/test/command/4742.md
+++ b/test/command/4742.md
@@ -7,7 +7,7 @@ extension properly.
 ^D
 [ Header
     1
-    ( "non-ascii--raksmorgas" , [] , [] )
+    ( "non-ascii-warning-raksmorgas" , [] , [] )
     [ Str "non"
     , Space
     , Str "ascii"
@@ -29,7 +29,7 @@ so it survives...
 ^D
 [ Header
     1
-    ( "non-ascii-\65039-r\228ksm\246rg\229s" , [] , [] )
+    ( "non-ascii-warning-r\228ksm\246rg\229s" , [] , [] )
     [ Str "non"
     , Space
     , Str "ascii"
@@ -49,7 +49,7 @@ so it survives...
 ^D
 [ Header
     1
-    ( "non-ascii-\65039-r\228ksm\246rg\229s" , [] , [] )
+    ( "non-ascii-warning-r\228ksm\246rg\229s" , [] , [] )
     [ Str "non"
     , Space
     , Str "ascii"
